docs/user/metacat/source/configuration.rst

```diff
@@ -134,20 +91,24 @@ To edit the authentication configuration file:
    default location for this file is in /var/metacat/config/, but this path is configurable,
    so it may be elsewhere.
 
-      .. Tip::
-         If you cannot find the **metacat-site.properties** file, its location is stored in a
-         property named ``application.sitePropertiesDir`` inside the **metacat.properties** file,
-         which can be found in:
-         ``<tomcat_app_dir>/<context_dir>/WEB-INF/metacat.properties``
-         (where the <context_dir> is the application context, usually named "metacat".)
-         See :ref:`configuration-properties-overview` for details
+  Tip: If you cannot find the **metacat-site.properties** file, its location is stored in a
+  property named ``application.sitePropertiesDir`` inside the **metacat.properties** file,
+  which can be found in:
+
+  ::
+
+    <tomcat_app_dir>/<context_dir>/WEB-INF/metacat.properties
+
+    (where the <context_dir> is the application context, usually named "metacat".)
+
+  See :ref:`configuration-properties-overview` for details
 
 2. Once you have located **metacat-site.properties**, change the following properties appropriately
-   (or add them if they do not already exist)
+(or add them if they do not already exist)
 
   ::
-  
-    auth.administrators  - a semicolon-separated list of administrators' ORCID IDs
+
+    auth.administrators  - a semicolon-separated list of administrators' ORCID iDs
     auth.url             - the authentication server URL
     auth.surl            - the authentication secure server URL
     auth.file.path       - the authentication password file path
```

Comment:
remove these 3 - no longer relevant: 

- auth.url & auth.surl are for ldap
- auth.file.path is for file-based auth

Reply:
Thank you!